### ISSUE TITLE
Include a `path` argument in all main functions

### ISSUE THREAD

**Issue by mpadge** [etiennebacher/altdoc#21]
Everything currently defaults to `getwd()`, which may fail. It's much safer for all arguments to accept a `path` argument, defaulting to some version of `here::here()`, which I do in `r2readthedocs`. `here` only depends on `rprojroot`, and that has no dependencies.

**Comment by etiennebacher** [comment]
@mpadge I'm not sure I understood correctly. Does this mean that each time I use a path I have to use `file.path()`? For instance:
```
fs::dir_exists("docs")
```
would become:
```
fs::dir_exists(file.path(path, "docs"))
```

**Comment by mpadge** [comment]
Yep. But if you're using `fs`, then there's also `fs::path() `. And `fs` is wise because base path manipulations can have issues on Windows

**Comment by etiennebacher** [comment]
Hello @mpadge, I just merged a PR that adds the argument "path" using `fs::path_abs()` (defaulting to `here::here()` as in `r2readthedocs`). It makes the code a bit harder to read. Maybe there's a way to refactor it later, but for now I mostly wanted something that works on all platforms. There shouldn't be any issues on Windows now, but maybe you have a use case to check for this?
